src/routines/get_results.jl

```diff
@@ -91,42 +83,18 @@ function get_optimizer_log(op_m::OperationModel)
         @warn("SolveTime() property not supported by $(optimizer_log[:solver])")
         optimizer_log[:solve_time] = "Not Supported by $(optimizer_log[:solver])"
     end
-
     return optimizer_log
-
 end
 
-function _export_optimizer_log(optimizer_log::Dict{Symbol, Any}, path::String)
-    df = DataFrames.DataFrame(optimizer_log)
-
-    file_path = joinpath(path,"$(round(Dates.now(),Dates.Minute))-optimizer_log.feather")
-
-    Feather.write(file_path, df)
+# Function to create a dictionary for the time series of the simulation
 
-    return
+function get_time_stamp(op_model::OperationModel)
 
+    initial_time = PSY.get_forecast_initial_times(op_model.sys)[1]
+    interval = PSY.get_forecasts_resolution(op_model.sys)
+    range = collect(initial_time:interval:initial_time+ Dates.Hour(24))
```

Comment:
Why is this range hardcoded? the range values have to be read from the OperationModel. 

```range = collect(initial_time:interval:initial_time + horizon*interval)```